apps/alarmeta/widget.js

```diff
@@ -0,0 +1,37 @@
+(() => {
+  const alarms = require("Storage").readJSON("sched.json",1) || [];
+
+  function draw() {
+    const times = alarms.map(alarm => require("sched").getTimeToAlarm(alarm)).filter(a => a !== undefined);
+    const next = Math.min.apply(null, times);
+    if (next > 0 && next < 86400000) {
+      const hours = Math.floor((next % 86400000) / 3600000).toString();
+      const minutes = Math.floor(((next % 86400000) % 3600000) / 60000).toString();
+
+      g.reset(); // reset the graphics context to defaults (color/font/etc)
+      g.setFontAlign(0,0); // center fonts
+      g.clearRect(this.x, this.y, this.x+this.width-1, this.y+23);
+
+      // Use 'locale' module to get a shortened month name
+      // in the correct language
```

Comment:
You could use the new locale_utils, see https://github.com/espruino/BangleApps/pull/1784

Reply:
Leftover comment from the template in http://www.espruino.com/Bangle.js+Widgets